draft-ietf-gnap-core-protocol.md

```diff
@@ -3212,7 +3272,13 @@ If the client instance is asking for more limited access, the AS could determine
 has been granted to the client instance and return the more limited access rights immediately.
 If the grant request was previously in the _approved_ state, the AS could decide to remember the larger scale of access rights associated
 with the grant request, allowing the client instance to make subsequent requests of different
-subsets of granted access. The details of this are out of scope for this specification.
+subsets of granted access. The details of this processing are out of scope for this specification,
+but a one possible approach is as follows:
+
+1. A client instance requests access to A, and is granted by the RO. This results in an access token.
```

Comment:
Need to succinctly explain what A and B are

Reply:
I was trying to avoid something with built-in semantics, since "write" implies you'd probably also get "read". That's why the example uses the generic "A" and "B" variables for the access rights. We could just as easily use "Foo" and "Bar" here.